File: C++/Programming Basics with C++/07. Exercise - Nested Conditional Statement/simple_task/HotelRoom.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string month;
    int nights;
    cin >> month >> nights;

    double apartmentPrice = 0;
    double studioPrice = 0;

    if (month == "May" || month == "October") {
        studioPrice = 50 * nights;
        apartmentPrice = 65 * nights;
        if (nights > 7 && nights <= 14) {
            studioPrice *= 0.95;
        } else if (nights > 14) {
            studioPrice *= 0.7;
        }
    } else if (month == "June" || month == "September") {
        studioPrice = 75.2 * nights;
        apartmentPrice = 68.70 * nights;
        if (nights > 14) {
            studioPrice *= 0.8;
        }
    } else if (month == "July" || month == "August") {
        studioPrice = 76 * nights;
        apartmentPrice = 77 * nights;
    }

    if (nights > 14) {
        apartmentPrice *= 0.9;
    }

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << "Apartment: " << apartmentPrice << " lv." << endl;
    cout << "Studio: " << studioPrice << " lv." << endl;

    return 0;
}

File: C#/Programming Basics with C#/07. Nested Conditional Statements - Exercise/10. Hotel Room/HotelRoom.cs
using System;

class HotelRoom
{
    static void Main()
    {
        string mount = Console.ReadLine().ToLower();
        int numberOfNights = int.Parse(Console.ReadLine());
        double studioPrice = 0.0;
        double appartmentPrice = 0.0;
        if (mount == "may" || mount == "october")
        {
            studioPrice = numberOfNights * 50.0;
            appartmentPrice = numberOfNights * 65.0;
            if (numberOfNights > 7 && numberOfNights <= 14)
            {
                studioPrice = studioPrice - studioPrice * 0.05;
            }
            else if (numberOfNights > 14)
            {
                studioPrice = studioPrice - studioPrice * 0.30;
            }
        }
        else if (mount == "june" || mount == "september")
        {
            studioPrice = numberOfNights * 75.20;
            appartmentPrice = numberOfNights * 68.70;
            if (numberOfNights > 14)
            {
                studioPrice = studioPrice - studioPrice * 0.20;
            }
        }
        else if (mount == "july" || mount == "august")
        {
            studioPrice = numberOfNights * 76.0;
            appartmentPrice = numberOfNights * 77.0;
        }

        if (numberOfNights > 14)
        {
            appartmentPrice = appartmentPrice * 0.9;
        }
        Console.WriteLine($"Apartment: {appartmentPrice:F2} lv.");
        Console.WriteLine($"Studio: {studioPrice:F2} lv.");
    }
}